package.json

```diff
@@ -21,7 +21,8 @@
     "release:dry-run": "DRY_RUN=true grabthar-release",
     "postrelease": "npm run cdnify -- --commitonly",
     "flatten": "grabthar-flatten",
-    "prepublishOnly": "npm run validate-components-for-publish"
+    "prepublishOnly": "npm run validate-components-for-publish",
+    "cdn-check": "babel-node --config-file './babel.config.json' --ignore ./fake --plugins=transform-es2015-modules-commonjs ./scripts/cdnCheck.js"
```

Comment:
Do you need the `--config-file` and `--ignore` flags here?  I had to use them in `sdk-logos` because of (I think) a bug with `babel-node` and I'm curious if the same issue happens in `sdk-release`.  No worries if they're needed though!

Reply:
good call, was going to test that out.